client/package-lock.json

```diff
@@ -0,0 +1,18703 @@
+{
```

Comment:
We are using yarn-lock files now rather than package.lock.json. Please fix this. 

1. Delete your node_modules
2. Delete your package.json files
3. Yarn install on all the folders 
4. Commit your yarn-lock file.